sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -78,6 +79,13 @@ This table shows the relationship between SDK versions and supported API version
 |5.0.0 - Latest GA release (can be installed by removing the `--pre` flag)| 3.0
 |5.1.0b3 - Latest release (beta)| 3.0, 3.1-preview.2, 3.1-preview.3
 
+Note: API version 3.1-preview.3 should be used exclusively for the Healthcare Analysis and Batch Analysis services.  When creating a Cognitive Service or Text Analytics resource for accesing these services, be sure to use flags `--sku S` and one of the following regions for the `--region` flag.
```

Comment:
does this means that a user can't target 3.1-preview.3 for i.e. DetectLanguage operation?

Reply:
Yes, I touched on this above in @deyaaeldeen 's comment as well, but the reason is that the other services are not all using /v3.1-preview.3 in their controller paths.  Since the autogenerated code sets all of the endpoints for each method for every API version, the result I observed is that if I use v3.1-preview.3 as the default that all of the generated methods for the other services will attempt to call the endpoints with `/v3.1-preview.3/` in the path.  

@iscai-msft , do you know if there is a way around this for a future release?

Reply:
oh very weird. This is going to confuse our customers :( For future releases, our SDKs should only target one version at a time.
I'll wait on Izzy's reply to see if this is true for Python too and how to avoid it.

Reply:
Python's just doing what the swagger has us do, adding the API version (`v3.1-preview.3`) to the path. This is the case for all of the API versions we've done. The reason why we only have `v3.1-preview.3` for `analyze` and `health`, was because we believed that the service did not have this API version up for other endpoints at the time, @abhahn to confirm. This may no longer be the case

Reply:
Hmm weird. Maybe when Python started the work that was the case but weeks before we released the SDKs, the service had support for all endpoints in 3.1-preview.3.
It will be good to learn more about why this happened. Also, for future releases, we need to be careful about this and make sure that what we are releasing is in consistency with the latest swagger and the other languages.